tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -551,21 +573,17 @@ def _check_agent_log(self) -> bool:
                 log.info("There are no errors in the agent log")
                 return True
 
-            log_path: Path = self.context.log_path/f"CheckAgentLog-{self.context.environment_name}.log"
```

Comment:
My previous PR create a log for each environment, so now we do not need to create this extra CheckAgentLog-*.log, we can simply log those messages to the environment log.